### PR TITLE
adds a rnd kit to the free miners vendor

### PR DESCRIPTION
**Pull request by boodaliboo**
https://forums.yogstation.net/index.php?threads/give-the-free-miners-a-box-similar-to-the-free-golems-liberators-legacy-for-2-5k-mining-points.21901/
they wanted it
#### Changelog

:cl:  
rscadd: Added rnd kit to free miners vendor
/:cl:

### CODE REVIEW

**Comment by ynot01** [comment]
bood you glorious dent i love you

File: code/modules/mining/machine_vending.dm [hunk 1]
```diff
@@ -300,7 +300,8 @@
 		new /datum/data/mining_equipment("Toxin First-Aid Kit",			/obj/item/storage/firstaid/toxin,								600, VENDING_MEDS),
 		new /datum/data/mining_equipment("Stimpack Bundle",				/obj/item/storage/box/medipens/utility,							200, VENDING_MEDS),
 		new /datum/data/mining_equipment("Point Transfer Card", 		/obj/item/card/mining_point_card,								500, VENDING_MISC),
-		new /datum/data/mining_equipment("Space Cash",    				/obj/item/stack/spacecash/c1000,								2000, VENDING_MISC)
+		new /datum/data/mining_equipment("Space Cash",    				/obj/item/stack/spacecash/c1000,								2000, VENDING_MISC),
+		new /datum/data/mining_equipment("Nanotrasen Brand Rnd Kit",  	/obj/item/storage/box/rndboards,								2000, VENDING_TOOL)
```
Comment (ynot01) [inline code/modules/mining/machine_vending.dm]:
```suggestion
		new /datum/data/mining_equipment("Nanotrasen Brand Circuitry Kit",  	/obj/item/storage/box/rndboards,								2000, VENDING_TOOL)
```
'Rnd Kit' looks ugly as hell

**Comment by Mesalikepie** [comment]
Terrific PR

**Comment by Reed0506** [comment]
if the box does not contain autolathe parts, that could be an issue because they would not be able to actually use the box without going to the station. if it does, fantastic.

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**